codecov.yml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project: yes
+    patch: yes
+    changes: yes
```

Comment:
There might be a problem currently with codecov posting status updates currently, but in general those are useful (next to the CI statuses).